pkgs/servers/sql/postgresql/generic.nix

```
@@ -266,36 +291,18 @@ let
       # resulting LLVM IR isn't platform-independent this doesn't give you much.
       # In fact, I tried to test the result in a VM-test, but as soon as JIT was used to optimize
       # a query, postgres would coredump with `Illegal instruction`.
-      broken = (jitSupport && stdenv.hostPlatform != stdenv.buildPlatform)
-        # Allmost all tests fail FATAL errors for v12 and v13
-        || (jitSupport && stdenv.hostPlatform.isMusl && olderThan "14");
+      broken = jitSupport && stdenv.hostPlatform != stdenv.buildPlatform;
```

In general, platforms are not comparable, using `canExecute` function seems appropriate here.

```suggestion
      broken = jitSupport && stdenv.hostPlatform.canExecute stdenv.buildPlatform;
```